.github/workflows/cloudrun-build.yml

```
@@ -1,67 +1,67 @@
-          gcloud run deploy ${{ env.SERVICE }} \
-            --image "$IMAGE_URI" \
-            --region ${{ env.REGION }} \
-            --platform managed \
-            --allow-unauthenticated \
-            --timeout 300s \
-            --set-env-vars "^@^ALLOWED_ORIGINS=${{ secrets.ALLOWED_ORIGINS }}@FIREBASE_PROJECT_ID=${{ env.PROJECT_ID }}@OPENAI_API_KEY=${{ secrets.OPENAI_API_KEY }}"
-
-      - name: 'Display service URL'
-        run: |
-          echo "Service deployed to:"
+name: Deploy Firebase Backend to Cloud Run
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)